database/migrations/2020_02_03_204857_create_availability_table.php

```diff
@@ -16,8 +16,10 @@ public function up()
         Schema::create('availability', function (Blueprint $table) {
             $table->bigIncrements('id');
             $table->unsignedInteger('user_id');
-            $table->timestamp('from');
-            $table->timestamp('to');
+            $table->timestamp('from')->nullable()
+                ->comment('Nullable to enforce compatibility with MYSQL strict mode.');
+            $table->timestamp('to')->nullable()
+                ->nullable('Nullable to enforce compatibility with MYSQL strict mode.');
```

Comment:
This should be a comment

Reply:
> If someone has availability, why would either from or to be null?

MySQL doesn't like letting in a zero value for timestamp by default, hence why this migration was failing with MySQL strict mode enabled.

Reply:
> This should be a comment

Fixed